### PR TITLE
Add -d --domain print only domain option

### PR DESCRIPTION
**Pull request by galeksandrp**

### CODE REVIEW

**Comment by john-kurkowski** [comment]
Can you give a description what inspired this? 

Can the same thing done with Taco Bell Programming? For example, piping to `cut`/`sed`.

File: tldextract/cli.py [hunk 1]
```diff
@@ -49,6 +51,14 @@ def main():
     elif len(args.input) is 0:
         parser.print_usage()
         exit(1)
+    elif args.domain:
+        for i in args.input:
+            ext = tld_extract(i)
+            if ext.suffix:
```
Comment (john-kurkowski) [inline tldextract/cli.py]: What is this checking?

File: tldextract/cli.py [hunk 2]
```diff
@@ -49,6 +51,14 @@ def main():
...
+                print(ext.registered_domain)
+            else:
+                print('.'.join([ext.subdomain.split('.')[-1], ext.domain]))
```
Comment (john-kurkowski) [inline tldextract/cli.py]: What is this building?

File: tldextract/cli.py (hunk 1 above)
Comment (galeksandrp) [inline tldextract/cli.py]: When `suffix` not present in PSL it contains empty string when `domain` contains actual notavalidsuffix, and `subdomain` contains sub.domains.domain.

**Comment by john-kurkowski** [comment]
Bump. You responded to 1 comment. Any thoughts on my 2 others?

**Comment by john-kurkowski** [comment]
Closing due to inactivity. I'm not sure what this feature is for. I'd consider it! Maybe [open an issue next time](https://www.igvita.com/2011/12/19/dont-push-your-pull-requests/).

**Pull request closed without merging**